SpecRunner.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
I don't think this file, and the lib/jasmine files, are necessary now that your Jasmine testing is being done through Node